### DISCUSSION TITLE
Recommended weather station?

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [jeroenterheerdt/HAsmartirrigation#375] [General]
I'm looking into buying a weather station to make this add-on more precise.
What is the recommended station/manufacturer to get?
Which station actually features a sensor for evapotranspiration?

**Comment by alexl04** [comment]
ECOWITT Wittboy Weather Station with Wireless Outdoor Sensor, 7 in 1 Weather Station with WiFi Hub, Improved GW2000 Gateway, Accurate Weather Forecast Data for Home Garden Farm 868MHz https://www.amazon.nl/dp/B09QKXMKS4?psc=1&ref_=cm_sw_r_apin_ct_6GFJ44TCFSE7W04RSJV9&language=en_US

ECOWITT WH40 Wireless Self Emptying Rain Collector Rain Sensor https://www.amazon.nl/dp/B09MHSSZS6?psc=1&ref_=cm_sw_r_apin_ct_DWJP9NYSDQYBGC3R59SG&language=en_US

**Comment by 4irm** [comment]
> I'm looking into buying a weather station to make this add-on more precise. What is the recommended station/manufacturer to get? Which station actually features a sensor for evapotranspiration?

This one is cheaper and rain measurement is much better...

https://www.amazon.de/dp/B0CGWVLCBP?psc=1&ref=ppx_pop_dt_b_product_details
